Libraries/Components/SafeAreaView/SafeAreaView.js

```diff
@@ -40,11 +40,22 @@ if (Platform.OS === 'android') {
 } else {
   const RCTSafeAreaViewNativeComponent = require('RCTSafeAreaViewNativeComponent');
   exported = class SafeAreaView extends React.Component<Props> {
```

Comment:
This should use forwardRef instead and assign the ref to the RCTSafeAreaViewComponent. Search the repo for forwardRef to check out other examples